packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -710,14 +710,57 @@ export class Scene {
       (acc, builder: SceneItemOperationsBuilder) => acc.concat(builder.build()),
       [] as QueryOperation[]
     );
-    return new ItemsOperationExecutor(
+    return new OperationExecutor(
       this.sceneViewId,
       this.stream,
       this.dimensions,
       operationList
     );
   }
 
+  /**
+   * Returns an executor that accepts a function as a parameter that contains one or more operations to apply
+   * to items or annotations in the scene view. The operations will be applied transactionally.
+   *
+   * @example
+   * ```typescript
+   * const viewer = document.querySelector('vertex-viewer');
+   * const scene = await viewer.scene();
+   *
+   * // Deselect everything, then select a specific item by ID
+   * await scene.elements(op => [
+   *   op.where(q => q.all()).deselect(),
+   *   op.where(q => q.withItemId('item-id')).select(),
+   * ]).execute();
+   * ```
+   *
+   * @see {@link RootQuery} for more information on available queries.
+   *
+   * @see {@link SceneItemOperationsBuilder} for more information on available operations to the scene items.
+   *
+   * @param itemOperations
+   */
+  public elements(
+    itemOperations: (q: SceneItemQueryExecutor) => TerminalItemOperationBuilder
```

Comment:
How do you envision this working on the PMI annotation side? Would it be a second function provided to the `elements()` function? Or did you have something else in mind?

Wondering if there might be a way to organize this so that it's more clear which data sets you're operating on. Just a thought, but maybe the `RootQuery` (or a new concept, but effectively the object passed to the `where` method of the `QueryExecutor`) could have internal definitions for both, so you'd have something like:

```typescript
const scene = await viewer.scene();

scene.elements(op => [
  op.where(q => q.items.withItemId(...)).select(),
  op.where(q => q.annotations.withAnnotationId(...)).select(),
])
```